doc/start/kube-helm.rst

```diff
@@ -0,0 +1,344 @@
+================================
+Installation (Kubernetes + Helm)
+================================
+
+The ceph-helm_ project enables you to deploy Ceph in a Kubernetes environement.
+This documentation assumes a Kubernetes environement is available.
+
+Current limitations
+===================
+
+ - The public and cluster networks must be the same
+ - If the storage class user id is not admin, you will have to manually create the user
+   in your Ceph cluster and create its secret in Kubernetes
+ - ceph-mgr can only run with 1 replica
+
+Install and start helm
+======================
+
+Helm can be installed by following these instructions_.
+
+Once installed, run::
+
+    $ helm init
+    $ helm serve &
+    $ helm repo add local http://localhost:8879/charts
+
+Add ceph-helm to Helm local repos
+==================================
+::
+
+    $ git clone https://github.com/ceph/ceph-helm
+    $ cd ceph-helm/ceph
+    $ make
+
+Configure your Ceph cluster
+===========================
+
+Create a ``~/ceph-overrides.yaml`` that will contain your Ceph configuration::
+
+    $ cat ~/ceph-overrides.yaml
+    network:
+      public:   172.21.0.0/20
+      cluster:   172.21.0.0/20
+
+    osd_devices:
+      - name: dev-sdd
+        device: /dev/sdd
```

Comment:
probably a stupid question, what about hostname or IP? where are these thing created?

Reply:
Do you mean the nodes that will hold these devices? If so they are defined via label assignment

Reply:
Thanks yes that is what i was looking for.